docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,31 @@
+# Troubleshooting
+
+If you are having problems with your AmpliPi device start here
+
+## Audio
+
+#### Noise
+Audio noise can have a lot of causes, below are some common reasons:
+* **Mixing grounds**: AmpliPi has two different grounded lines, analog and digital. The most common way to mix them is by plugging in an audio device and attempting to power it by the usb on the rear of the unit, all usb-powered devices must be powered from a wall outlet
+* **Speaker setup**: Some speaker configurations can lead to excess noise being inducted into the output. This is unrelated to the system and has everything to do with how the speakers are wired and installed.
+* **Hardware issues**: See the support page for more info.
+
+## Updating
+
+1. Go to the web app at http://amplipi.local/
+2. Click the gear icon (**⚙**) in the bottom right corner to go to the configuration page
+3. Select **Updates** and click the **Check for Updates** button
+
+## New Release
+To update your AmpliPi to the latest version:
+
+4. Click the **Update** button
+
+## Prereleases and using previous versions
+4. Click the **Older Releases** tab and select the release you would like to use
+5. Click the **Start Update** button
+
+## Reimaging AmpliPi
+To bring AmpliPi's OS and install back to its original state see: https://amplipi.discourse.group/t/re-imaging-the-raspberry-pi/46
+
+![https://amplipi.discourse.group/t/re-imaging-the-raspberry-pi/46](imgs/manual/reimaging-your-pi-qr.svg)
```

Comment:
I would much prefer that the QR code and link here point to [Klayton's new Balena Etcher docs](https://github.com/micro-nova/AmpliPi/blob/1e18ac7852a3340caacbe0e4b938336e7ce4d6fc/docs/imaging_etcher.md).